### PR TITLE
Build /cloud/multi-cloud page

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Build /cloud/multi-cloud page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Please check page against [copy doc](https://docs.google.com/document/d/1nWsDaRiUzaupjKDzu1ZqjlmxHGv2HCgW8oBBEacYFLs/edit#) and [design](https://app.zeplin.io/project/6141d0b4743dc928767d815e/screen/6149c055f798ec581c0f5431).
- [Demo](https://ubuntu-com-10466.demos.haus/cloud/multi-cloud)

## Issue / Card

Fixes [#4416](https://github.com/canonical-web-and-design/web-squad/issues/4416)

## Screenshots
What is multi-cloud? | Ubuntu ()

### CODE REVIEW

**Comment by long-chung** [comment]
Hi Min, LGTM.
Just one minor comment. Could you tighten up the spacing in the "Multi-cloud vs Hybrid cloud" section between the h5 and paragraphs. i.e "Is every hybrid cloud a multi-cloud?" should be .5rem bottom margin.

Could you also do this in the "What is hybrid cloud" page as well please under the "Hybrid cloud vs multi-cloud" section

File: templates/cloud/multi-cloud.html [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+        }}
+      </div>
+      <h4>Application management software</h4>
+      <p class="u-sv1">Enables <a href="https://juju.is" class="p-link--external">deploying, integrating and managing</a> Kubernetes, container and VM-native applications seamlessly across various cloud platforms.</p>
+    </div>
+    <p>
+      When combined with the <a href="/cloud/hybrid-cloud">hybrid cloud architecture</a>, multi-cloud can be also integrated with the HCM software to enable dynamic placement of the workloads depending on capacity and cost conditions.
+    </p>
+  </div>
+  </div>
```
Comment (bethcollins92) [inline templates/cloud/multi-cloud.html:304]: There is a stray end div here 
```suggestion

```

**Review by bethcollins92 — APPROVED** [review]
Looking good Min! Just remember to add the link back to the page which pointed to this one (was it on the hybrid cloud page?).

File: templates/cloud/multi-cloud.html (hunk 1 above)
Comment (bethcollins92) [reply templates/cloud/multi-cloud.html:304]: Oh and there's a conflict to fix in the `navigation.yaml` also

**Pull request merged**